components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1922,7 +1923,7 @@ void AdsServiceImpl::ShowNotification(
 #endif
 }
 
-bool AdsServiceImpl::ShouldShowNotifications() const {
+bool AdsServiceImpl::ShouldShowNotifications() {
```

Comment:
de-const'ing?

Reply:
and all the other de-const's - presuming a valid reason but curious...

Reply:
Unfortunately, some of the calls could no longer be const due to changing of state